tcms/rpc/tests/test_testexecution.py

```diff
@@ -37,6 +37,34 @@ def test_add_comment_with_pk_as_int(self):
         self.assertEqual(created_comment['comment'], first_comment.comment)
 
 
+class TestExecutionRemoveComment(APITestCase):
+    """Test TestExecution.remove_comment"""
+
+    def _fixture_setup(self):
+        super()._fixture_setup()
+
+        self.execution = TestExecutionFactory()
+
+    def test_delete_all_comments_with_pk(self):
+        self.rpc_client.TestExecution.add_comment(self.execution.pk, "Hello World!")
+        self.rpc_client.TestExecution.add_comment(self.execution.pk, "More comments")
+        self.rpc_client.TestExecution.remove_comment(self.execution.pk)
+
+        comments = get_comments(self.execution)
+        self.assertEqual(comments.count(), 0)
+
+    def test_delete_one_comment_with_pk(self):
+        self.rpc_client.TestExecution.add_comment(self.execution.pk, "Hello World!")
+        comment_2 = self.rpc_client.TestExecution.add_comment(self.execution.pk, "Cool comment")
+
+        self.rpc_client.TestExecution.remove_comment(self.execution.pk, comment_2['id'])
+        comments = get_comments(self.execution)
+
+        first_comment = comments.first()
+        self.assertEqual("Hello World!", first_comment.comment)
+        self.assertEqual(comments.count(), 1)
```

Comment:
you can swap the assertion order. In case there is an error you'll get a mismatch in the expected number. Otherwise first_comment may be None and you'll get a weird NoneType has no attribute named comment error.